docs/contributing/introduction-to-contributing.md

```diff
@@ -66,9 +69,13 @@ See [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for m
 
    - The title must begin with "feat:", "fix:", or anything related to your changes. <br /> **Tips**: You can follow your chosen option when [committing](#commits) your changes.
 
-   - Unsolicited code is welcomed, but an issue is required to announce your intentions. <br /> **_PRs without a linked issue will be marked invalid and closed_**.
-  
-> [!IMPORTANT]  
+   - Unsolicited code is welcomed, but an issue is required to announce your intentions.
+
+⚠️ **PRs will be marked as invalid and may be closed if:**
+   - the issue is not assigned to the contributor who opened the PR
+   - no issue is linked to the PR
```

Comment:
Please clean the format to follow Markdown best prattice as below:

```suggestion
⚠️ **PRs will be marked as invalid and may be closed if:**

- the issue is not assigned to the contributor who opened the PR
- no issue is linked to the PR
```

Reply:
In my recent commit I've done this change